context-monitoring/src/main/java/de/atb/context/services/wrapper/AmIMonitoringDataRepositoryServiceWrapper.java

```diff
@@ -49,95 +46,74 @@ public AmIMonitoringDataRepositoryServiceWrapper(final IAmIMonitoringDataReposit
      *
-     * @see
-     * IMonitoringDataRepository#
-     * getLastIds(de.atb.context.common.util.ApplicationScenario,
-     * java.lang.Class, de.atb.context.common.util.TimeFrame)
+     * @see IMonitoringDataRepository#getLastIds(de.atb.context.common.util.ApplicationScenario, java.lang.Class, de.atb.context.common.util.TimeFrame)
      */
     @Override
     public final synchronized List<String> getLastIds(final ApplicationScenario applicationScenario, final Class<Type> clazz, final TimeFrame timeFrame)
         throws ContextFault {
         return service.getLastIds(applicationScenario, clazz.getName(), timeFrame);
     }
 
-    /*
+    /**
      * (non-Javadoc)
      *
-     * @see
-     * IMonitoringDataRepository#
-     * getLastIds(de.atb.context.common.util.BusinessCase, java.lang.Class,
-     * de.atb.context.common.util.TimeFrame)
+     * @see IMonitoringDataRepository#getLastIds(de.atb.context.common.util.BusinessCase, java.lang.Class, de.atb.context.common.util.TimeFrame)
      */
     @Override
     public final synchronized List<String> getLastIds(final BusinessCase businessCase, final Class<Type> clazz, final TimeFrame timeFrame) throws ContextFault {
         return service.getLastIds(businessCase, clazz.getName(), timeFrame);
     }
 
-    /*
+    /**
      * (non-Javadoc)
      *
-     * @see
-     * IMonitoringDataRepository#
-     * getMonitoringData(de.atb.context.common.util.ApplicationScenario,
-     * java.lang.Class, int)
+     * @see IMonitoringDataRepository#getMonitoringData(de.atb.context.common.util.ApplicationScenario, java.lang.Class, int)
      */
     @Override
     public final List<Type> getMonitoringData(final ApplicationScenario applicationScenario, final Class<Type> clazz, final int count) throws ContextFault {
         List<String> modelStrings = service.getMonitoringData(applicationScenario, clazz.getName(), count);
         return convertStringsToModels(modelStrings, clazz);
     }
 
-    /*
+    /**
      * (non-Javadoc)
      *
-     * @see
-     * IMonitoringDataRepository#
-     * getMonitoringData(de.atb.context.common.util.BusinessCase,
-     * java.lang.Class, int)
+     * @see IMonitoringDataRepository#getMonitoringData(de.atb.context.common.util.BusinessCase, java.lang.Class, int)
      */
     @Override
     public final List<Type> getMonitoringData(final BusinessCase businessCase, final Class<Type> clazz, final int count) throws ContextFault {
         List<String> modelStrings = service.getMonitoringData(businessCase, clazz.getName(), count);
         return convertStringsToModels(modelStrings, clazz);
     }
 
-    /*
+    /**
      * (non-Javadoc)
      *
-     * @see
-     * IMonitoringDataRepository#
-     * getMonitoringData(de.atb.context.common.util.BusinessCase,
-     * java.lang.Class, de.atb.context.common.util.TimeFrame)
+     * @see IMonitoringDataRepository#getMonitoringData(de.atb.context.common.util.BusinessCase, java.lang.Class, de.atb.context.common.util.TimeFrame)
```

Comment:
shouldn't there be : `* @see IMonitoringDataRepository#getMonitoringData(de.atb.context.common.util.ApplicationScenario, java.lang.Class, de.atb.context.common.util.TimeFrame)`?

Reply:
changed